docs/reference/installation.rst

```diff
@@ -28,9 +28,6 @@ Add the dependant bundles to the vendor/bundles directory:
 .. code-block:: bash
 
     composer require sonata-project/page-bundle --no-update
-
-    # for SonataPageBundle > 2.3.6
-    composer require sonata-project/datagrid-bundle --no-update
```

Comment:
[datagrid-bundle is already required in composer.json](https://github.com/sonata-project/SonataPageBundle/blob/3.x/composer.json#L32)